plugins/ucrm-client-signup/src/includes/initialize.php

```diff
@@ -0,0 +1,13 @@
+<?php
+# SDK Data Variables
+$configManager = \Ubnt\UcrmPluginSdk\Service\PluginConfigManager::create();
+$config = $configManager->loadConfig();
+
+$optionsManager = \Ubnt\UcrmPluginSdk\Service\UcrmOptionsManager::create();
+$options = $optionsManager->loadOptions();
+
+## Just a unique key to give to ember for extra security when making requests
+// $key = password_hash($options->pluginPublicUrl.PROJECT_PATH, PASSWORD_DEFAULT); // This does not work
+$key = "this_key_should_be_improved";
```

Comment:
I would save some generated key to the data directory when there is none and use that. To safely generate random data, you can use for example `base64_encode(random_bytes(48))`